com/win32comext/mapi/NOTICE.md

```diff
@@ -0,0 +1,6 @@
+# Licensing
+
+This distributed package contains components under multiple licenses:
+
+- `mapi.pyd` is licensed under the MIT License. See [com/win32comext/mapi/src/MAPIStubLibrary/LICENSE](https://github.com/mhammond/pywin32/blob/main/com/win32comext/mapi/src/MAPIStubLibrary/LICENSE) for details.
```

Comment:
MIT being a well-known license with a dedicated SPDX identifier hopefully this should be enough?

Reply:
I've also added it as `MAPI-LICENSE.txt`